src/templates/next/components/InfoCards/InfoCards.tsx

```diff
@@ -0,0 +1,116 @@
+import { InfoCardsProps } from "~/common"
+import Card from "../shared/Card"
+
+const TitleSection = ({
+  title,
+  subtitle,
+  className,
+}: {
+  title: InfoCardsProps["title"]
+  subtitle: InfoCardsProps["subtitle"]
+  className?: string
+}) => {
+  return (
+    <div className={`flex flex-col gap-8 self-start max-w-3xl ${className}`}>
+      <h3 className="text-content text-2xl leading-tight font-semibold sm:text-4xl ">
+        {title}
+      </h3>
+      {subtitle && (
+        <p className="text-content text-sm sm:text-lg">{subtitle}</p>
+      )}
+    </div>
+  )
+}
+
+const InfoCards = ({
+  sectionIdx,
+  cards,
+  title,
+  subtitle,
+  variant,
+}: InfoCardsProps) => {
+  const bgColor = sectionIdx % 2 === 1 ? "bg-gray-100" : "bg-white"
+  return (
+    <section className={`py-24 px-5 lg:px-10 ${bgColor}`}>
+      {variant === "side" ? (
+        <div
+          className={`flex flex-col lg:flex-row gap-12 items-center mx-auto lg:max-w-5xl`}
+        >
+          <TitleSection
+            title={title}
+            subtitle={subtitle}
+            className="lg:max-w-60"
+          />
+          <div
+            className={`grid grid-cols-1 lg:grid-cols-3 gap-8 md:hidden lg:grid`}
+          >
+            {cards.map((card) => (
+              <Card
+                title={card.title}
+                imageUrl={card.imageUrl}
+                text={card.text}
+                imageAlt={card.imageAlt}
+                buttonLabel={card.buttonLabel}
+                buttonUrl={card.buttonLabel}
+                variant="vertical"
+              ></Card>
+            ))}
+          </div>
+          <div
+            className={`hidden grid-cols-1 lg:grid-cols-3 gap-8 md:grid lg:hidden`}
```

Comment:
same comment about unnecessary `lg:grid-cols-3` 
and using `md-max-lg`

Reply:
https://github.com/isomerpages/isomer-components/pull/38/commits/b9090a42c9eb867b02b08434c37283ed65251b38